### PR TITLE
Added Frames, UnitTests, Strongname signing

### PR DESCRIPTION
**Pull request by spreedated**
A whole lot of changes:

- Added signed assembly
- Added UnitTesting (NUnit)
- Added FrameStyles (pipe & double pipes)
- Updated wiki
- Changed delimiter and dividerstring from "string" to "char"
- some cleanups

### CODE REVIEW

**Comment by minhhungit** [comment]
@spreedated thank you for you PR

- One stupid question can you explain please, what and why we need strongname signing
- Aslo your test project is using **.net 5**, could you pls modify to support net35
- The original purpose I used string instead char for delimiter is I want to allow developers enter any thing they want, but it looks it's too wordy, so it is okay to change it to char 👍 
- FrameStyle option as I understand is the box of the table, but I see that it does not support other formats like markdown, alternative... so do you think we should add it as new formation, like ConsoleTableBuilderFormat.FramePip or ConsoleTableBuilderFormat.FrameDoublePip instead of creating new FrameStyle attribute 

Thank you

**Comment by spreedated** [comment]
- Strongnamed projects needs all referenced assemblies to be strongnamed as well. We can either provide it or let the developers do it by downloading and compiling the source.
- I can change to multitargeting frameworks
- For the FrameStyle, I think your idea of an option to create a map is good - and convert the current styles into maps as well, so it's easier to implement

**Comment by spreedated** [comment]
For the signing, the file is an SNK, so it's not secured by a password. That's fine, developers can still debug and build without a password.

**Comment by minhhungit** [comment]
>Strongnamed projects needs all referenced assemblies to be strongnamed as well. We can either provide it or let the developers do it by downloading and compiling the source.

can they install it from nuget without strong name ?

Btw, I will be very happy to approve this pull request after you convert your FrameStyles to Formats

Thank you again

**Comment by spreedated** [comment]
Yes you can download it.
Here's a description from Microsoft https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named

I get asap into the changes, probably tomorrow :)

**Comment by minhhungit** [comment]
@spreedated  Sorry for rushing this a bit, but have you had a chance to update this sir

I also pushed code for char map feature in this branch https://github.com/minhhungit/ConsoleTableExt/tree/Feature/BoxDrawingChars

I want to merge your pull request first before I merge BoxDrawingChars to master

**Comment by minhhungit** [comment]
Can't wait, I merged BoxDrawingChars branch and release version 3.0.0 with charmap and many things. I will close this PR, feel free to create new one, many thanks @spreedated
